src/woo_search/search_index/tests/test_search_api.py

```diff
@@ -1382,21 +1518,25 @@ def test_filter_by_publisher_and_information_category_and_result_type(self):
             uuid="dd3b8be0-e461-498a-9a18-0f5fc8adc956",
             publisher=publisher_1,
             informatie_categorieen=[ic_1],
+            onderwerpen=[topic_1],
```

Comment:
I wouldn't do this, considering that most won't have topics.

See the comment above too - you're adding a dimension (from 3 to 4), which results in 2^4 possible combinations = 16. That's a lot more combinations/test cases to handle.